script.js

```diff
@@ -264,3 +264,26 @@ form.addEventListener('submit', (btn) => {
     form.reset();
   }
 });
+
+// Local Storage
+
+const fullName = form.elements.name;
+const emailAddress = form.elements.email;
+
+// Access stored form data if present
+const storedData = JSON.parse(localStorage.getItem('formData'));
+if (storedData) {
+  fullName.value = storedData.name;
+  emailAddress.value = storedData.email;
+}
+
+// Save formData
+function saveData() {
+  const formData = {
+    name: fullName.value,
+    email: emailAddress.value,
+  };
```

Comment:
- [ ] Good job so far 👍🏽. Kindly ensure that all the contact information is stored in local storage. Currently, you are not storing the data from the `<textarea> `to local storage. Please add it also 😄 

e.g
```

const formData = {
    name: fullName.value,
    email: emailAddress.value,
    body: userMessage.value
  };
```